native-imagetranscoder/src/main/jni/native-imagetranscoder/jpeg/jpeg_stream_wrappers.cpp

```diff
@@ -133,6 +133,11 @@ static void osInitDestination(j_compress_ptr cinfo) {
 
   // allocate java byte array
   dest->javaBuffer = env->NewByteArray(kStreamBufferSize);
+  if (dest->buffer == NULL) {
```

Comment:
shouldn't this be dest->javaBuffer? How did you verify your changes?

Reply:
@oprisnik Updated.